src/go/rpk/pkg/cli/cluster/config/get.go

```diff
@@ -40,10 +40,31 @@ output, use the 'edit' and 'export' commands respectively.`,
 			client, err := adminapi.NewClient(fs, p)
 			out.MaybeDie(err, "unable to initialize admin client: %v", err)
 
-			currentConfig, err := client.Config(cmd.Context(), true)
+			currentConfig, err := client.SingleKeyConfig(cmd.Context(), key)
 			out.MaybeDie(err, "unable to query current config: %v", err)
 
 			val, exists := currentConfig[key]
+			if !exists {
+				// Here, we deal with the possibility that "key" is an alias for a property name.
+				// If len(currentConfig) > 1, there is likely a mismatch between
+				// the rpk version and redpanda, since support for cluster config
+				// get requests with single keys was only added post v24.2.1.
+				if len(currentConfig) > 1 {
+					out.Die(`Get request for property '%s' returned multiple values - likely an alias.
+                                                 Re-request the original property name, or ensure redpanda and rpk versions
+                                                 are both up to date.`, key)
```

Comment:
This whitespace isn't trimmed when printing, it needs to be removed here OR change each line of this to be wrapped with `"text goes here\n" + `

Reply:
Removed this section of dead code, but thank you for the heads up!